languages/python/oso/tests/test_polar.py

```diff
@@ -752,3 +762,79 @@ def kwarg_method(self, x=1, y=2):
     qvar("kwargs(result)", "result") == [3, 4]
     qvar("args(result)", "result") == [5, 6]
     qvar("mixed(result)", "result") == [7, 8]
+
+
+def unwrap_and(x):
+    assert isinstance(x, Expression)
+    assert x.operator == "And"
+    if len(x.args) == 1:
+        return x.args[0]
+    else:
+        return x.args
+
+
+def test_partial(polar):
```

Comment:
Can you have more than one partial in a query? Can those partials reference each other?
```python
f(a,b) if a.x = b.y
```
```python
polar.query_rule("f", (Partial("A"), Partial("B"))
```

Reply:
Super extra bonus points for somehow coming up with a join in the django integration.

Reply:
You can have multiple partials, and they can reference each other, but it doesn't currently work on the way out to Python. Agree that a join is the right interpretation.